examples/E2EBert/news_classifier.py

```diff
@@ -300,9 +339,10 @@ def training_step(self, train_batch, batch_idx):
         :return: output - Training loss
         """
         input_ids = train_batch["input_ids"]
-        attention_mask = train_batch["attention_mask"]
+        input_ids = input_ids.to(device)
```

Comment:
Can you please check if `.to(device)` is needed when using PTL

Reply:
@shrinath-suresh  I belive we need this anyway I will cross check